packages/backend/src/peripherals/database/AggregatedReportRepository.ts

```diff
@@ -181,10 +180,6 @@ export class AggregatedReportRepository extends BaseRepository {
   async addOrUpdateMany(reports: AggregatedReportRecord[]) {
     const rows = reports.map(toRow)
     const knex = await this.knex()
-    const timestampsMatch = reports.every((r) =>
```

Comment:
Why this change?

Reply:
This is no longer needed and would make tests unnecessary call addOrUpdate for every entry seperately, I can change if you want to, but this assert is useless IMO